server/src/main/resources/application.properties

```diff
@@ -10,8 +10,8 @@ spring.datasource.password = ${ELO_TRACKER_DB_PASSWORD}
 # The SQL dialect makes Hibernate generate better SQL for the chosen database
 spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.MySQL8Dialect
 
-# Hibernate ddl auto (create, create-drop, validate, update)
+# Hibernate ddl auto (createMatch, createMatch-drop, validate, updateMatch)
 spring.jpa.hibernate.ddl-auto = create-drop
```

Comment:
Create Drop is dangerous. Are we using a MYSQL db.

Reply:
We are, I had it at create-drop while i was getting set up, we should probably set it to update